WD-Firmware/demo/build/demos/demo_comrv_baremetal.py

```diff
@@ -27,8 +32,18 @@ def __init__(self):
         'D_TICK_TIME_MS=4',
         'D_ISR_STACK_SIZE=400',
     ]
-      
+
     self.listSconscripts = [
       'comrv_baremetal',
-      'psp'
-    ]
\ No newline at end of file
+      'psp',
+      'demo_comrv_baremetal',
+    ]
+
+    self.listDemoSpecificCFlags = [
+      '-fcomrv',
+    ]
+
+    self.listDemoSpecificLinkerFlags = [
+      '-Wl,--grouping-file=' + self.strGrpFile,
+      '-Wl,--defsym=__comrv_cache_size=' + self.strComrvCacheSize

```

Comment:
add comment to explain -grouping-file. this is user flag, we want them to understand --defsym=__comrv_cache_size

Reply:
done in caec55c